docs/source/conf.py

```diff
@@ -26,7 +26,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "SQLFluff"
-copyright = "2019, Alan Cruickshank"
+copyright = "2023, Alan Cruickshank"
```

Comment:
Would it make sense to always use the current year?

Reply:
Probably - Seemed more legit to update it manually rather than have it automated in some way.